### ISSUE TITLE
ValueError: too many values to unpack (expected 5)

### ISSUE THREAD

**Issue by gtors** [SectorLabs/django-postgres-extra#71]
Django 2.1.2
Python 3.7
Psqlextra 1.20 (also reproduced in 1.21a12)

```
class MyModel(PostgresModel):
    id = models.BigAutoField(primary_key=True)
    code = models.CharField(max_length=16)
    name = models.CharField(max_length=256)

MyModel.objects.values_list('pk', flat=True)
```

```
Traceback (most recent call last):
  File "<console>", line 1, in <module>
  File "/home/gtors/.local/share/virtualenvs/tko-5nB7BPJA/lib/python3.7/site-packages/django/db/models/query.py", line 777, in values_list
    clone = self._values(*_fields, **expressions)
  File "/home/gtors/.local/share/virtualenvs/tko-5nB7BPJA/lib/python3.7/site-packages/django/db/models/query.py", line 745, in _values
    clone.query.set_values(fields)
  File "/home/gtors/.local/share/virtualenvs/tko-5nB7BPJA/lib/python3.7/site-packages/django/db/models/sql/query.py", line 1987, in set_values
    self.add_fields(field_names, True)
  File "/home/gtors/.local/share/virtualenvs/tko-5nB7BPJA/lib/python3.7/site-packages/psqlextra/query.py", line 116, in add_fields
    _, targets, _, joins, path = self.setup_joins(parts, opts, alias, allow_many=allow_m2m)
```

`self.setup_joins(parts, opts, alias, allow_many=allow_m2m)` returns tuple with 6 items:

```
(Pdb) self.setup_joins(parts, opts, alias, allow_many=allow_m2m)
JoinInfo(final_field=<django.db.models.fields.BigAutoField: id>, targets=(<django.db.models.fields.BigAutoField: id>,), opts=<Options for MyModel>, joins=['core_mymodel'], path=[], transform_function=<function Query.setup_joins.<locals>.final_transformer at 0x7f1b34034d90>)
```

`transform_function` was added in this commit https://github.com/django/django/commit/2162f0983de0dfe2178531638ce7ea56f54dd4e7

**Comment by Cabalist** [comment]
This appears to be fixed in #66 
Could we get that merged?

**Comment by thinkwelltwd** [comment]
This is still broken for me, on Djanog 2.1, Python3.6
